Ginger/GingerCoreNET/Run/GingerExecutionEngine.cs

```diff
@@ -416,6 +416,13 @@ public void RunRunner(bool doContinueRun = false)
             bool runnerExecutionSkipped = false;
             try
             {
+                for (int i = 0; i < BusinessFlows.Count; i++)
```

Comment:
instead of puting this and other checks for skipped in different places in flow, add on Runner Finally, before reporting on RunnerEnd a call to a method which will look for all Skipped BF/ActivitiesGroup/Activities and report each to Logger